File: src/test/resources/js/InlineMathOperations.js
class InlineMathOperations {
    complexCalculation(a, b) {
        const sum = a + b;
        const product = a * b;
        const difference = a - b;
        return sum + product - difference;
    }

    calculateResult1(x, y) {
        return this.complexCalculation(x, y) + 10;
    }

    calculateResult2(p, q) {
        return 2 * this.complexCalculation(p, q);
    }
}

File: src/test/resources/py/InlineMathOperations.py
class InlineMathOperations:

	def complexCalculation(self, a, b):
		sum_ = a + b
		product = a * b
		difference = a - b
		return sum_ + product - difference

	def calculateResult1(self, x, y):
		return self.complexCalculation(x, y) + 10

	def calculateResult2(self, p, q):
		return 2 * self.complexCalculation(p, q)
